Files/Dialogs/SettingsDialog.xaml.cs

```diff
@@ -10,6 +11,9 @@ public sealed partial class SettingsDialog : ContentDialog
     {
         public SettingsViewModel AppSettings => App.AppSettings;
 
+        // for some reason the requested theme wasn't being set on the content dialog, so this is used to manually bind to the requested app theme
+        FrameworkElement RootAppElement => Window.Current.Content as FrameworkElement;
```

Comment:
Dos this work when the theme is changed at run time?

Reply:
yes